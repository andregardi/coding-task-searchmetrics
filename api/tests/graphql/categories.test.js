const { createTestClient } = require('apollo-server-testing');
const apolloServer = require('../../src/graphql/apolloServer');
const categoriesQuery = require('./__queries__/categories');

describe('categories', () => {
  it('The categories query should match the snapshot', async () => {
    const { query } = createTestClient(apolloServer);
    const res = await query({
      query: categoriesQuery,
    });
    expect(res).toMatchSnapshot();
  });
});
