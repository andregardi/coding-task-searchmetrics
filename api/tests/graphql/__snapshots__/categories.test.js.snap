// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`categories The categories query should match the snapshot 1`] = `
Object {
  "data": Object {
    "categories": Array [
      Object {
        "keywords": Array [
          "audi",
          "bmw",
          "tires",
        ],
        "name": "cars",
      },
      Object {
        "keywords": Array [
          "bianchi",
        ],
        "name": "bikes",
      },
      Object {
        "keywords": Array [
          "banana",
          "avocado",
        ],
        "name": "fruit",
      },
      Object {
        "keywords": Array [
          "cat",
          "dog",
          "otter",
        ],
        "name": "animals",
      },
      Object {
        "keywords": Array [
          "tea",
          "water",
        ],
        "name": "drinks",
      },
    ],
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
